ssbarnea/ansible-zuul-jobs
# fix/github workflows

- Make ansible-navigator-jobs strict (#633)
- Utils: Migrate integration tests jobs to network ee (#1089)
- Add irrelevant-files to some jobs (#1101)
- Remove ansible-navigator-jobs (#1100)
- okd: don't use k.core 1.2 (#1102)
- okd: use 3pci-check pipeline (#1104)
- Clean up network-ee integration tests (#1106)
- Add back vyos.vyos /usr/bin/python3 (#1109)
- Setup FIPS mode for network-ee integration jobs (#1105)
- add periodical jobs to satisfy ansible inclusion (#1111)
- Revert "vmware: add freebsd13.iso (#994)" (#1112)
- split_targets/test get_all_targets (#1103)
- Enable sanity tests for community.okd (#1110)
- Add aws-complete pipeline which runs all integration tests in collection (#1071)
- cloud.common: periodic jobs need the artifacts (#1114)
- Switch to network-ee-integration-tests-ios jobs (#1040)
- Add test command specific pip requirements/constraints (#991)
- Add publish-to-automation-hub-3pci (#1121)
- Pin OKD to specific k8s.core version (#1119)
- ansible-test-sanity-docker: bump to py38 (#1125)
- govcsim: run on python38 (#1127)
- Bump to Python 3.8 AWS (#1130)
- Move from python3.6 to python3.8 for eos devel jobs (#1133)
- changetopy38 (#1134)
- Bump to Python 3.8 k8s (#1131)
- Use Python 3.8 for NX-OS devel jobs (#1132)
- Use py38 (#1135)
- [Meta Collection] add ansible.network network ee jobs (#1092)
- Move netcommon-specific test runs to python38 (#1139)
- Add eos stable-2.12 jobs (#1142)
- Add Vyos stable2.12 jobs (#1141)
- Add ansible-test-sanity-docker-stable-2.12 for vyos  (#1144)
- UTILS: Add stable-2.12 jobs for utils (#1147)
- Add ios stable-2.12 integration test jobs (#1143)
- add stable 2.12 jobs for junos (#1149)
- Ansible Network Stable 2.12 Jobs Added (#1150)
- [ovs] Add stable-2.12 jobs (#1146)
- Add stable-2.12 jobs for NX-OS (#1145)
- [frr] Add stable-2.12 sanity job (#1152)
- Add ansible-test-sanity-docker-stable-2.12 for iosxr (#1148)
- asa py38 job update (#1154)
- support 2.12 (#1155)
- Add fedora-34 for ansible-tox-linters  (#1157)
- Enable periodic jobs for trendmicro.deepsec (#1159)
- CentOS: use Facebook mirror instead of Vexxhost (#1163)
- move ansible-galaxy-importer on centos-8-stream-small (#1161)
- Add community.crypto to AWS deps so we can generate SSL certs (#1165)
- drop the asav9-12-3-python37 nodeset (#1162)
- configure-mirrors-fork/yum: retries 3 times on failre (#1172)
- govcsim: ensure py38 is installed (#1153)
- Added network-ee-tests-stable-2.12 job (#1140)
- ansible-test-sanity-docker: test milestone (#1171)
- a-t-molecule-kubernetes-core: test milestone and devel (#1170)
- move ansible-test-sanity-docker on f34 (#1169)
- AWS tests - make 'devel' sanity non-voting and add stable-2.12 (#1167)
- Replaced centos-8-1vcpu label with centos-8-stream (#1164)
- support 2.12 (#1158)
- project-templates.yml: add stable-2.12 branch to templates (#1166)
- cloud.common: temp turn k8s check non votin (#1176)
- vmware-ci-nfs-share: fetch ISO from s3 (#1115)
- ensure py38 for aws integration jobs (#1179)
- centos8/yum update, retry until is not failed (#1186)
- To enable py 26, 35 and 36 jobs for TM to fulfill inclusion criteria (#1175)
- ansible-ee-tests-base: retry the container pull (#1191)
- Revert "cloud.common: temp turn k8s check non votin (#1176)" (#1177)
- Drop py37 in favor for py38 (#1108)
- publish-to-automation-hub-3pci: use the right pipeline (#1124)
- cloud.common: avoid dup jobs (#1192)
- Remove py35 unit test (#1189)
- Migrate eos integration test jobs to network-ee (#1193)
- remove the jinja2<3.0.0 constaint (#1200)
- Enable FIPS mode for eos libssh jobs (#1198)
- aws integration tests: use ansible milestone branhc (#1199)
- [python3.9] Add ansible-tox-py39 job (#1201)
- Fix check for py39 on fedora 34/35 (#1205)
- Add eos periodic jobs (#1206)
- Add vyos periodic jobs (#1216)
- Move the govcsim jobs on Fedora 35 (#1211)
- vmware/integration: install openssl on the controller (#1217)
- community.vmware uses vmware_rest during the tests (#1215)
- bump the vmware nodeset to fedora-35 (#1210)
- vmware: use vSphere 7.0.3 (#1207)
- vcenter: install squid on the datastore (#1113)
- ansible-test-base: install the right Python version (#1223)
- split_targets: Allow to run unstable test (#1203)
- update fedora-lastest-1vcpu to Fedora 35 (#1209)
- run controller-python37 on Fedora 35 (#1219)
- Revert "run controller-python37 on Fedora 35 (#1219)" (#1227)
- avoid role name clash with ensure-python (#1226)
- Replace ubuntu-bionic and fedora-34 with fedora-35 (#1222)
- Remove integration jobs from periodic check (#1229)
- Add py38 unit test for vyos (#1232)
- Add periodic jobs for NX-OS (#1230)
- Add periodic jobs for OVS (#1231)
- update periodic jobs (#1238)
- Add periodic jobs for FRR (#1239)
- replace selinux with selinux-please-lie-to-me on Fedora (#1236)
- Add periodic jobs for utils (#1233)
- add periodic jobs (#1240)
- Add periodic jobs for iosxr and remove 2.9 jobs from ansible for iosxr (#1234)
- ansible-test: retry once the pip install of the req (#1241)
- Set py27 unit test to non-voting (#1237)
- To fix TM deepsec py27, 35 and 36 jobs (#1224)
- remove unused integration job (#1245)
- remove fedora-32/SELinux work around (#1244)
- remove playbooks/ansible-cloud/py36/pre.yaml (#1243)
- vmware: ensure SPS is running (#1235)
- restore playbooks/ansible-cloud/py38/pre.yaml (#1248)
- playbooks/ansible-cloud/py38/pre.yaml is not required anymore (#1247)
- vcenter: use become to run service-control (#1249)
- Make network-ee test for utils non-voting (#1251)
- add periodic jobs for IOS (#1253)
- Remove .github check from github-workflows job
